addons/website_hr_recruitment/models/hr_applicant.py

```diff
@@ -25,3 +25,14 @@ def website_form_input_filter(self, request, values):
             if stage:
                 values['stage_id'] = stage.id
         return values
+
+    def _notify_get_recipients_groups(self, message, model_description, msg_vals=None):
```

Comment:
Don't understand why this is in "website_hr_recruitment" and not in "hr_recruitment"?

Reply:
As per the current flow the button appears when the website_hr_recruitment is installed so that why we have done that part here.